pkg/apis/eventing/v1alpha1/trigger_defaults_test.go

```diff
@@ -67,6 +70,34 @@ func TestTriggerDefaults(t *testing.T) {
 				},
 				Spec: TriggerSpec{Broker: otherBroker, Filter: defaultTriggerFilter}},
 		},
+		"subscriber, ns defaulted": {
```

Comment:
why updating the `v1alpha1` stuff ?

Reply:
it is a non-api breaking change, so updating v1alpha1 resources works out

Reply:
Yeah, not doing it is more work (and I think more confusing), and it's non-breaking.